src/webgl/shaders/lighting.glsl

```diff
@@ -33,8 +33,16 @@ uniform float uConstantAttenuation;
 uniform float uLinearAttenuation;
 uniform float uQuadraticAttenuation;
 
+// setting from  _setImageLightUniforms()
+// boolean to initiate the calculateImageDiffuse and calculateImageSpecular
+uniform bool uUseImageLight;
+// storing the texture
+uniform sampler2D equiRectangularTextures;
+
 const float specularFactor = 2.0;
 const float diffuseFactor = 0.73;
+const float PI = 3.141;
```

Comment:
Can this be used from another common constant file? Would be better to have a value closer to real PI, since 3.14159 is closer to 3.142

Reply:
I don't think we have a file for constants because mostly constants are taken via the "MATH" object but it isn't available in shader languages according to my knowledge.  

We can do `#define PI 3.141592` at the top of this file.
Another possible option is to do  `#define PI 3.141592` in the `\webgl\shaders\webgl2Compatibility.glsl` file.

I am not sure which one is better.

Reply:
For now since we only use this here and we don't use pi anywhere else in a shader yet (to my surprise, haha) I think we can keep it in this file for now, and if we find we need to use it in another file too (maybe we need it for the specular shader?) we can factor it out into a new shader constants file at that point